input/cql/AgeStratificationExample.cql

```diff
@@ -0,0 +1,26 @@
+library AgeStratificationExample version '1.0.0'
+
+using FHIR version '4.0.1'
+
+codesystem "ISO-8601-Derived Periods": 'http://terminology.hl7.org/CodeSystem/iso-8601-derived-periods' // { 'P0Y--P1Y', 'P1Y--P5Y', ... }
+
+// Age Groups
+code "P10Y--P14Y": 'P10Y--P15Y' from "ISO-8601-Derived Periods" display '10-14 years'
```

Comment:
The code identifier (`"P10Y--P14Y"`) needs to match the code (`'P10Y--P15Y'`), please update